hera_qm/xrfi.py

```diff
@@ -1099,37 +1299,48 @@ def xrfi_run(ocalfits_file, acalfits_file, model_file, data_file, history,
 
```

Comment:
Even though we don't return anything, I think we should document all of the files that we write out, either in a `Notes` section or the function description.